src/GogoKit/Models/Response/Event.cs

```diff
@@ -13,6 +13,9 @@ public class Event : EmbeddedEvent
         [DataMember(Name = "min_ticket_price")]
         public Money MinTicketPrice { get; set; }
 
+        [DataMember(Name = "number_of_tickets")]
+        public int NumberOfTickets { get; set; }
```

Comment:
Thanks for the PR @brunoss! Can you please make the property `int?`? We make value types on response models nullable since there's no guarantee they will be returned in response JSON. e.g. The client could use our Sparse Fieldsets feature (see http://developer.viagogo.net/#sparse-fieldsets) to return a partial object